toolchains/jdk_build_file.bzl

```
@@ -83,6 +83,9 @@ java_runtime(
         ":jdk-lib",
         ":jre",
     ],
+    # Provide the 'java` binary explicitly so that the correct path is used by
+    # Bazel even when the host platform differs from the execution platform.
+    java = glob(["bin/java.exe", "bin/java"])[0],
```

It'd be nice if this worked with https://github.com/bazelbuild/bazel/issues/8195
```suggestion
    java = glob(["bin/java.exe", "bin/java"], allow_empty = True)[0],
```

---

Is every individual pattern checked for emptiness?

---

Looks like it.

---

I also filed https://github.com/bazelbuild/rules_java/issues/134